ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -22,40 +22,268 @@ resources:
+      PODS_PER_NODE: "100"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
+      REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - ref: openshift-qe-node-density
+    workflow: rosa-aws-sts-hypershift
+- as: ocp-qe-perfscale-ci-tests-rosa-node-density-heavy
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      PODS_PER_NODE: "100"
+      REPLICAS: "3"
+    test:
+    - ref: openshift-qe-node-density-heavy
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-node-density-heavy-24nodes
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      PODS_PER_NODE: "100"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - ref: openshift-qe-node-density-heavy
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-hcp-node-density-heavy-24nodes
   cron: 0 12 * * 4
   steps:
     cluster_profile: aws-qe
     env:
       CHANNEL_GROUP: nightly
       COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
       ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
       MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.13"
+      PODS_PER_NODE: "100"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
       REPLICAS: "24"
+      ZONES_COUNT: "3"
     test:
-    - ref: openshift-qe-cluster-density-v2
+    - ref: openshift-qe-node-density-heavy
+    workflow: rosa-aws-sts-hypershift
+- as: ocp-qe-perfscale-ci-tests-rosa-network-perf
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      REPLICAS: "3"
+    test:
+    - chain: openshift-qe-network-perf
     workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-network-perf-24nodes
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-network-perf
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-hcp-network-perf-24nodes
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
+      REPLICAS: "24"
```

Comment:
for network perf tests I believe number of replicas doesnt too much of an impact. Joe had lowered the number of replicas compared to all other replica counts for a given profile in the configuration here: https://github.com/openshift/release/pull/40587
@jtaleric what are your thoughts on this set up/number of worker node replicas?

Reply:
Yes, network-perf does not need a scaled cluster. The default number of workers should be fine. CC: @SachinNinganure

Reply:
The default will result in 1 node per region, which isn't enough -- unless we want to test throughput across AZs. 

I recommend we go with 2 nodes per AZ, so 6 total worker nodes.

Reply:
@jtaleric Hi Joe, do you mean in network-perf with should test within AZ but not across AZ? e.g. node1(pod on node1) on AZ1 -> node2(pod on nodes) on AZ2?